modules/ballerina-builtin/src/main/ballerina/ballerina/caching/caching.bal

```diff
@@ -53,8 +53,8 @@ public function createCache (string name, int expiryTimeMillis, int capacity, fl
         error e = {msg:"Capacity must be greater than 0."};
         throw e;
     }
-    // Cache eviction factor must be between 0.0 and 1.0 inclusive.
-    if (evictionFactor < 0 || evictionFactor > 1) {
+    // Cache eviction factor must be between 0.0 (exclusive) and 1.0 (inclusive).
+    if (evictionFactor <= 0 || evictionFactor > 1) {
         error e = {msg:"Cache eviction factor must be between 0.0 and 1.0 inclusive."};
```

Comment:
The error message must be updated with the updated rule; i.e. 0.0 is not inclusive now.